File: Lecture 3/lecture3_src/scores2.c
// Среднее значение трёх чисел с использованием массивов

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Получение входных значений, введённых пользователем
    int scores[3];
    scores[0] = get_int("Score: ");
    scores[1] = get_int("Score: ");
    scores[2] = get_int("Score: ");

    // Вывод среднего значения
    printf("Average: %f\n", (scores[0] + scores[1] + scores[2]) / 3.0);
}

File: Lecture 7/lecture7_src/2/scores2.py
# Среднее значение трёх чисел с использованием списка и цикла с оператором +

from cs50 import get_int

# Инициализация массива scores и его элементов
scores = []
for i in range(3):
    score = get_int("Score: ")
    scores += [score]

# Вывод среднего значения
average = sum(scores) / len(scores)
print(f"Average: {average}")
